src/anemoi/inference/runner.py

```diff
@@ -647,16 +647,27 @@ def forecast_stepper(
         is_last_step : bool
             True if it's the last step of the forecast
         """
-        steps = lead_time // self.checkpoint.timestep
+        rollout_step = self.checkpoint.timestep * self.checkpoint.multi_step_output
```

Comment:
Very minor thing but I think calling this rollout step is a bit confusing? I would maybe call it `output_step`